readthedocs/restapi/views/footer_views.py

```diff
@@ -81,7 +81,7 @@ def footer_html(request):
     version = get_object_or_404(
         Version.objects.public(
             request.user, project=project, only_active=False),
-        slug=version_slug)
+        slug__iexact=version_slug)
```

Comment:
We also seem to be doing a silly thing when slugifying versions, not creating _real_ slugs: https://github.com/rtfd/readthedocs.org/blob/fix-footer/readthedocs/builds/version_slug.py#L83

Reply:
We should probably use the SlugField that comes with django, https://docs.djangoproject.com/en/1.9/ref/models/fields/#slugfield

Reply:
Or autoslugfield from django-annoying

Reply:
Yea, I worry about changing slugification in this PR. I think it will be reasonably complex, and require a migration of all pre-existing slugs, along with the served documentation (since it will change the URL).

Reply:
Yeah. Good point.

Is it a good idea to create an issue to switch to one of this SlugFields, linking this and comment?

Reply:
+1